modules/client-handler/src/main/java/org/apache/ignite/client/handler/requests/sql/ClientSqlProperties.java

```diff
@@ -68,6 +75,10 @@ SqlProperties toSqlProps() {
             builder.set(QueryProperty.DEFAULT_SCHEMA, schema);
         }
 
+        if (timeZoneId != null) {
+            builder.set(QueryProperty.TIME_ZONE_ID, ZoneId.of(timeZoneId));
+        }
```

Comment:
How are we going to specify or reference the timezone format in other clients? Just give a reference to `java.time.ZoneId` javadocs?

Reply:
Yes, the javadoc explains it in detail: [ZoneId#of](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/ZoneId.html#of(java.lang.String))

It supports standard [IANA region-based zone IDs](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones), so I expect every client to have something suitable in the standard library or a popular package.